keras_cv/metrics/coco/__init__.py

```diff
@@ -13,5 +13,12 @@
 # limitations under the License.
 
 from keras_cv.metrics.coco.mean_average_precision import COCOMeanAveragePrecision
-from keras_cv.metrics.coco.pycoco_wrapper import PyCOCOWrapper
-from keras_cv.metrics.coco.pycoco_wrapper import compute_pycoco_metrics
+
+try:
+    from keras_cv.metrics.coco.pycoco_wrapper import PyCOCOWrapper
```

Comment:
This seems risky.  Can we instead do this on object instantiation?
```
def get_coco_eval():
    try:
        import cocoeval
        return cocoeval
     except e:
          raise ValueError(...)
```


then in the class:
```
cocoeval = get_coco_eval()
```

Reply:
I don't think this will work, because we are defining a class which is a subclass of `COCO`.
Maybe there's a way to do this which I'm not aware of, but as far as I can tell, we can't define that class if the import fails.